docs/qix/README.md

```diff
@@ -18,14 +18,16 @@ enigma.js can connect to any supported version of Qlik Sense by using the schema
         - [Dimensions](dimensions.md)
         - [Measures](measures.md)
         - [Variables](variables.md)
+- [Instances](instances.md)
```

Comment:
This is intentional. Should `instances` really be a sub-section of
- [Global]
    - [Apps]
?